langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -59,6 +69,19 @@ public transactional isolated function info() returns Info = @java:Method {
 
 # Returns information about the transaction with the specified xid.
 #
+# ```ballerina
+# byte[] xid = [48,53,101,102,101,55];
+#
+# transaction {
+#     transaction:Info? info = transaction:getInfo(xid);
+#     check commit;
+# }
+#
+# transactional function transFunc() returns error? {
+#    transaction:Info? info = transaction:getInfo(xid);
+# }
```

Comment:
Non-transactional functions can be called in a normal function too. So, I think we don't need the transaction block/transactional function and multiple examples here.

Reply:
My intention is to show multiple usages of `getInfo()`.  Doesn't the user want to know these  2 usages?

Reply:
If they can use it anywhere, why do we need to show these two scenarios specifically?

Reply:
Ah Now I got your point